lib_eio/process.ml

```diff
@@ -61,12 +61,12 @@ module Pi = struct
 
   module type MGR = sig
     type tag
-    type t
+    type t 
```

Comment:
@Arogbonlo please look at your PR yourself before submitting it for review. This change is obviously not supposed to be here.